### DISCUSSION TITLE
show-status图片无法加载问题

### DISCUSSION THREAD

**Discussion by caoxuexi** [TaleLin/lin-ui#1251] [求助] [answered]
当我这么使用status-show的时候，type填了data，应该是能找到data.png显示的，但是调试器中显示无法找到
<l-status-show show="{{!isOK}}" describe="订单中包含有售罄/下架/超出数量的商品" type="data"></l-status-show>
**调试器中报错如下：**
[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/status-show/image/data.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error)

**Comment by juzi214032** [comment]
lin ui 版本是多少

**Reply by caoxuexi** [reply]
最新0.9.2

**Reply by juzi214032** [reply]
开发者工具版本呢 windows还是 mac

**Reply by caoxuexi** [reply]
1.03.2012121也是最新，调试基础库2.12和最新的2.14.3都试过，都是同样的报这个错

**Reply by caoxuexi** [reply]
windows

**Reply by juzi214032** [reply]
那运行源码里的examples会报错吗

**Reply by juzi214032** [reply]
可以试一下跑example代码，如果这个没问题，那应该就是你代码的问题，可以再贴你代码看一下

**Comment by juzi214032** [comment, marked as answer]
**确认问题为微信开发者工具的问题，回退到 1.03.2011120 Stable 版本可正常使用**

1.03.2011120 Stable 下载地址：
- [Windows](https://dldir1.qq.com/WechatWebDev/release/p-ae42ee2cde4d42ee80ac60b35f183a99/wechat_devtools_1.03.2011120_x64.exe)
- [Mac OS](https://dldir1.qq.com/WechatWebDev/release/p-ae42ee2cde4d42ee80ac60b35f183a99/wechat_devtools_1.03.2011120.dmg)
